tasks/__init__.py

```diff
@@ -504,6 +519,10 @@ def test_travis_else():
     karma(single=True, browsers='PhantomJS')
 
 @task
+def test_travis_varnish():
```

Comment:
@sloria This one is here because it matches the pattern in travis. There's a test_travis_osf  and a test_travis_else too. I can remove test_varnish if you think that's worth it.